fera_full.py

```
@@ -1,3 +1,189 @@
+                    Anexo_Eletronico_Pdfs P 
+                    '''
+                    try:
+                        cursor.custom_execute(select_all_pdfs, None, True, False)
+                        relats = cursor.fetchall()
+                        notindexed.append(pathp)
+                        for rel in relats:                                   
+                            notindexed.append(os.path.join(global_settings.pathdb.parent, rel[2]))
+                        if(len(notindexed)>0): 
+                            tupleinfo = indexador_fera.addrels('relatorio', pathpdfinput = notindexed, pathdbext=sys.argv[1], \
+                                                                   rootx=global_settings.root, sqliteconnx=sqliteconn)
+                    except sqlite3.OperationalError as ex:  
```

Ruff

Local variable `ex` is assigned to but never used

Remove assignment to unused variable `ex`

(F841)